src/sentry/utils/snuba.py

```diff
@@ -858,6 +865,53 @@ def _snql_query(params: Tuple[SnubaQuery, Hub, Mapping[str, str]]) -> RawResult:
         raise SnubaError(err)
 
 
+def _legacy_snql_query(params: Tuple[SnubaQuery, Hub, Mapping[str, str]]) -> RawResult:
```

Comment:
Nit: Does `_snql_legacy_query` make more sense, since we try snql first?

Reply:
I decided to leave this, since the function takes a legacy query and outputs a snql query.